### ISSUE TITLE
Crippling "outdated repository" warning

### ISSUE THREAD

**Issue by DavidRodriguezSoaresCUI** [MiKTeX/miktex#738]
First of all, I'm sorry if this is a solved issue (or if this is not the place). I searched for a while with no answer so I filed this issue as well as I could.

[x] I have read and acknowledged this HOWTO article

I just installed Miktex on my Windows 10 x64 (version 20H2) machine with the basic installer from https://miktex.org/download (options : install just for me, always ask to fetch missing packages). I need it to convert a Jupyter notebook to PDF through Visual Studio Code. Each time I try converting, the process starts, a Miktex window asking to get a missing resource from repository "ftp://ftp.tu-chemnitz.de/pub/tex/systems/win32/miktex/tm/packages/", I click "Install", the window disappears and the process errors out (the error message is not specific).

What I tried :
 - Opening Miktex Console and clicking on "Check for update" : I get a "Something went wrong while checking for updated. Do you want to see the error details?" dialog box. If I accept, a larger dialog box contains the error and I can copy the error log (see below). Miktex Console doesn't crashes.
 - Opening Miktex Console and trying to manually install package from "Packages tab" : I get the same larger dialog box from before, same message, same log.

TL;DR: An error message cripples Mixtex usability.
 
Note : For privacy reasons I edit any mention of my username in provided logs, replaced by "***".

Error log:
===========================================================
```GENERAL MIKTEX INFORMATION
ReportDate: 2021-03-10 19:47:56
CurrentVersion: 21.2
SetupDate: 2021-03-10 18:02:00
SetupVersion: 21.2
Configuration: Regular
GitInfo: 7a7e645 / 2021-02-20 10:38:14
OS: Windows 10 Home, 64-bit, build 19042
SharedSetup: no
LinkTargetDirectory: C:\Users\***\AppData\Local\Programs\MiKTeX\miktex\bin\x64
PathOkay: yes
LastUpdateCheck: not yet
LastUpdate: not yet
LastUpdateDb: not yet
SystemAdmin: yes
RootPrivileges: no
AdminMode: no
Root0: C:\Users\***\AppData\Roaming\MiKTeX
Root1: C:\Users\***\AppData\Local\MiKTeX
Root2: C:\Users\***\AppData\Local\Programs\MiKTeX
Root3: C:\ProgramData\MiKTeX
Root4: C:\Program Files\MiKTeX
UserInstall: C:\Users\***\AppData\Local\Programs\MiKTeX
UserConfig: C:\Users\***\AppData\Roaming\MiKTeX
UserData: C:\Users\***\AppData\Local\MiKTeX
CommonInstall: C:\Program Files\MiKTeX
CommonConfig: C:\ProgramData\MiKTeX
CommonData: C:\ProgramData\MiKTeX
Invokers: non-existing/explorer

The following issues were detected:
  1: major issue: So far, you have not checked for MiKTeX updates.

ERROR DETAILS
Program: C:\Users\***\AppData\Local\Programs\MiKTeX\miktex\bin\x64\miktex-console.exe
Source: Libraries\MiKTeX\PackageManager\RestRemoteService.cpp:288
Error: The remote package repository is outdated. You have to choose another repository.
Details: 
  url: ftp://ftp.tu-chemnitz.de/pub/tex/systems/win32/miktex/tm/packages/
```

I checked the log under "%LOCALAPPDATA%\MiKTeX\miktex\log" and the only errors are "Something is wrong with the PATH", "minor issue: The PATH variable does not include the MiKTeX executables" and "The remote package repository is outdated. You have to choose another repository."

About the "PATH" issue : (after a restart) I checked my user's PATH and it contains both :
 - C:\Users\***\AppData\Local\Programs\MiKTeX\miktex\bin\x64\
 - C:\Users\***\AppData\Local\Programs\MiKTeX 2.9\miktex\bin\x64\
So I think these errors are resolved
 
I don't know if I want to provide all logs because they contain privacy sensitive information. I include both logs that have WARN/ERROR/FATAL statements, with pryvacy edits.

I suppose I still can resolve missing packages by downloading/installing them manually but that's really not ideal.

Questions : Is there a workaround to avoid this error ?

Attachments :
[xelatex.log](https://github.com/MiKTeX/miktex/files/6118549/xelatex.log)
[miktex-console.log](https://github.com/MiKTeX/miktex/files/6118550/miktex-console.log)
